catalog/ui/search-ui/search-ui-app/pom.xml

```diff
@@ -46,28 +46,27 @@
                     </execution>
                 </executions>
             </plugin>
+            <!-- Puts the features XML file for this app into the maven repo. -->
             <plugin>
-                <groupId>org.apache.karaf.tooling</groupId>
-                <artifactId>features-maven-plugin</artifactId>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>build-helper-maven-plugin</artifactId>
                 <executions>
                     <execution>
-                        <id>create-kar</id>
+                        <id>attach-configs</id>
+                        <phase>package</phase>
                         <goals>
-                            <goal>create-kar</goal>
+                            <goal>attach-artifact</goal>
                         </goals>
                         <configuration>
-                            <!-- Workaround to prevent features.xml file from being included twice in the kar file. -->
-                            <resourcesDir>${project.build.directory}/doesNotExist</resourcesDir>
-                            <featuresFile>${basedir}/target/classes/features.xml</featuresFile>
+                            <artifacts>
+                                <artifact>
+                                    <file>src/main/resources/org.codice.ddf.ui.searchui.filter.RedirectServlet.config</file>
+                                    <type>config</type>
+                                    <classifier>redirect</classifier>
```

Comment:
Just curious. Couldn't find any information about `config` type and `redirect` classifier. How did you know you needed to use those?